include/cutlass/conv/threadblock/conv2d_fprop_filter_tile_access_iterator_optimized.h

```diff
@@ -166,11 +170,14 @@ class Conv2dFpropFilterTileAccessIteratorOptimized{
     CUTLASS_PRAGMA_UNROLL
     for (int s = 0; s < ThreadMap::Iterations::kStrided; ++s) {
       uint32_t pred = ((column + s * ThreadMap::Delta::kStrided < problem_size_.K) ? 1u : 0);
-      predicates_ |= (pred << s);
+      CUTLASS_PRAGMA_UNROLL
+      for (int v_idx = 0; v_idx < kAccessesPerVector; ++v_idx) {
+        predicates_[v_idx] |= (pred << s);
+      }
     }
 
-    if (filter_c_ >= problem_size.C) {
-      predicates_ = 0u;
+    for (int v_idx = 0; v_idx < kAccessesPerVector; ++v_idx) {
```

Comment:
you neeed `CUTLASS_PRAGMA_UNROLL` here

Reply:
We need `CUTLASS_PRAGMA_UNROLL` before every for loop.  It is essential for the compiler to map the fragments/arrays to the registers.  Otherwise, the compiler would choose to use local memory which will slow the performance by a lot.